invenio_cli/cli.py

```diff
@@ -107,68 +53,51 @@ def init(flavour):
 
 
 @cli.command()
-@click.option('--local/--containers', default=True, is_flag=True,
-              help='Which environment to build, it defaults to local')
 @click.option('--pre', default=False, is_flag=True,
               help='If specified, allows the installation of alpha releases')
 @click.option('--lock/--skip-lock', default=True, is_flag=True,
               help='Lock dependencies or avoid this step')
-def build(pre, local, lock):
-    """Either setups project locally or builds container images.
+def install(pre, lock):
+    """Installs the  project locally.
 
-    --local option installs dependencies, creates instance directory,
+    Installs dependencies, creates instance directory,
     links invenio.cfg + templates, copies images and other statics and finally
     builds front-end assets.
-
-    --containers creates docker images inside of which the equivalent of
-    --local has been run.
     """
     cli_config = CLIConfig()
-    commands = Commands(cli_config, local)
-    commands.build(pre=pre, lock=lock)
+    commands = LocalCommands(cli_config)
+    commands.install(pre=pre, lock=lock)
 
 
 @cli.command()
-@click.option('--local/--containers', default=True, is_flag=True,
-              help='Which environment to build, it defaults to local')
 @click.option('--force', default=False, is_flag=True,
               help='Force recreation of db tables, ES indices, queues...')
 def services(local, force):
```

Comment:
minor: I understood from the readme and what we talked on Gitter, that services won't need `--local`/`--containers`?

Reply:
Fixed (removed the `local` parameter). Yes, `invenio-cli services` would be enough and would just spin up the services needed in development. `invenio-cli containerize` would start them also if not running AND start the other production-like containers.